cpp/dolfinx/fem/FiniteElement.cpp

```diff
@@ -427,7 +423,8 @@ bool FiniteElement::interpolation_ident() const noexcept
   return _element->interpolation_is_identity();
 }
 //-----------------------------------------------------------------------------
-xt::xtensor<double, 2> FiniteElement::interpolation_points() const
+std::pair<std::vector<double>, std::array<std::size_t, 2>>
```

Comment:
Same question as above, should this rather be a pair of std::span and an std::array, to avoid copying?